docusaurus.config.js

```diff
@@ -7,8 +7,7 @@ const UIKitReferencePlugins = require('./plugins/ui-kit-reference-plugin.cjs');
 const { webpackPlugin } = require('./plugins/webpack-plugin.cjs');
 const posthogPlugin = require('./plugins/posthog-plugin.cjs');
 
-/** @type {import('@docusaurus/preset-classic').Options} */
-const defaultSettings = {
```

Comment:
is this a typo or the `const` declaration not required anymore?

Reply:
Ah, my bad. It might've gotten removed by mistake.